src/routes/categoryRoutes.ts

```diff
@@ -0,0 +1,101 @@
+import express, { Request, Response } from 'express';
+import Categories from '../models/categories';
+import { v4 as uuidv4 } from 'uuid';
+import getCurrentTimeStamp from '../utils/currentTimeStamp';
+
+const router = express.Router();
+
+// * Create Category
+router.post('/categories', async (req: Request, res: Response) => {
+  const { title, description, image } = req.body;
+
+  try {
+    const category_id: string = uuidv4();
+    const created_at: Date = getCurrentTimeStamp();
+
+    const createCategory = new Categories({ category_id, title, description, image, created_at, isDeleted: false });
```

Comment:
- check if the category is already exists
- title should be unique across the collection